manifests/j/jumppad/jumppad/0.5.49/jumppad.jumppad.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: jumppad.jumppad
+PackageVersion: 0.5.49
+Installers:
+- InstallerUrl: https://github.com/jumppad-labs/jumppad/releases/download/v0.5.49/jumppad_0.5.49_windows_x86_64.zip
+  Architecture: x86
```

```suggestion
  Architecture: x64
```
`x86` validations are performed on a 32-bit OS, so I'm guessing that this is a 64-bit program.

---

Hey thanks, this is actually a mistake carried over from our first submission, will fix